cmd/explored/node.go

```diff
@@ -48,6 +50,8 @@ func newNode(addr, dir string, c consensus.Checkpoint) (*node, error) {
 	cm := chain.NewManager(chainStore, tip.Context)
 	tp := txpool.New(tip.Context)
 	cm.AddSubscriber(tp, cm.Tip())
+	e := explorer.New(tip.Context, nil)
+	cm.AddSubscriber(e, cm.Tip())
```

Comment:
this error needs to be checked, and it should use the *explorer's* tip, not the chain manager's tip. `AddSubscriber` tries to "migrate" the subscriber from its current tip to `cm.Tip()`, by calling `ProcessChainApplyUpdate` and `ProcessChainRevertUpdate` as needed. So if you pass `cm.Tip()`, no work is necessary (and thus no error is possible) -- but in most cases, we want to pass the subscriber's tip.